ti-sgx-ddk-km: properly handle tq-linux-gnueabi compiler prefix

From: Martin Schmiedel <Martin.Schmiedel@tq-group.com>

The additional patch for ti-sgx-ddk-km remaps the arm-oe-linux-gnueabi prefix
to the arm-linux-gnueabihf preferred target. Same should be done for the
arm-tq-linux-gnueabi prefix.

Signed-off-by: Martin Schmiedel <Martin.Schmiedel@tq-group.com>
---
 .../eurasiacon/build/linux2/config/compiler.mk     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eurasia_km/eurasiacon/build/linux2/config/compiler.mk b/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
index 97b90e2..50088af 100644
--- a/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
+++ b/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
@@ -68,7 +68,7 @@ define calculate-compiler-preferred-target
   ifneq ($$(filter aarch64-%,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := aarch64-linux-gnu
   endif
-  ifneq ($$(filter arm-%-gnueabihf arm-oe-linux-gnueabi arm-poky-linux-gnueabi,$$($(1)_compiler_preferred_target)),)
+  ifneq ($$(filter arm-%-gnueabihf arm-oe-linux-gnueabi arm-poky-linux-gnueabi arm-tq-linux-musleabi arm-tq-linux-gnueabi,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := arm-linux-gnueabihf
   endif
   ifneq ($$(filter arm-%-gnueabi armv7a-cros-linux-gnueabi armv7hl-redhat-linux-gnueabi,$$($(1)_compiler_preferred_target)),)
